sdk/src/utils/test.rs

```diff
@@ -102,7 +102,7 @@ pub(crate) fn static_test_v1_uuid() -> &'static str {
 /// Creates a claim for testing (v2)
 pub fn create_test_claim() -> Result<Claim> {
     // First create and add a claim thumbnail (we don't need to reference this anywhere)
-    let mut claim = Claim::new("adobe unit test", Some("adobe"), 2);
+    let mut claim = Claim::new("adobe unit test", Some("contentauth"), 2);
```

Comment:
```suggestion
    let mut claim = Claim::new("contentauth unit test", Some("contentauth"), 2);
```

Do you want to update the other string too maybe, so it's consistent?